dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScope.kt

```diff
@@ -22,10 +26,14 @@ internal class RumSessionScope(
 
     internal val activeChildrenScopes = mutableListOf<RumScope>()
 
-    internal var sessionId = NULL_SESSION_ID
+    internal var applicationStart: Long = RumFeature.timeProvider.getDeviceTimestamp()
```

Comment:
I am wondering why we are not using the value from `Datadog#startupTimeNs`

Reply:
Good question. 
Mostly it was because the `startupTimeNs` is in Nanosecondes and we need milliseconds, and that I wrote this before creating the `startupTimeNs`.

Reply:
Maybe we should drop this in order to be consistent...now we are measuring `app-start-time` with one value for `start` and we are setting the `start-time` from a different place.

Reply:
Fixde 👍